infantry:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100

  # Position Controllers --------------------------------------
  Rev27_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev27
    pid: {p: 1.0, i: 0.01, d: 0.01}
  Rev28_position_controller:
    type: velocity_controllers/JointVelocityController
    joint: Rev28
    pid: {p: 10.0, i: 1, d: 1}
  Rev29_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev29
    pid: {p: 1.0, i: 0.01, d: 0.01}
  Rev30_position_controller:
    type: velocity_controllers/JointVelocityController
    joint: Rev30
    pid: {p: 10.0, i: 1, d: 1}

  Rev31_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev31
    pid: {p: 1.0, i: 0.01, d: 0.01}
    
  Rev32_position_controller:
    type: velocity_controllers/JointVelocityController
    joint: Rev32
    pid: {p: 10.0, i: 1, d: 1}
  Rev33_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev33
    pid: {p: 1.0, i: 0.01, d: 0.01}
  Rev34_position_controller:
    type: velocity_controllers/JointVelocityController
    joint: Rev34
    pid: {p: 10.0, i: 1, d: 1}
  Rev35_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev35
    pid: {p: 0.001, i: 1, d: 0.4}
  Rev36_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev36
    pid: {p: 1.0, i: 0.01, d: 0.01}
  Rev37_position_controller:
    type: effort_controllers/JointPositionController
    joint: Rev37
    pid: {p: 1.0, i: 0.01, d: 0.01}
